ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml

```diff
@@ -60,6 +60,9 @@ gcc48_devtoolset_compiler:
   - devtoolset-2-binutils
   - devtoolset-2-gcc-c++
 
+Additional_Build_Tools_CentOS6:
```

Comment:
Can you add a comment to this (a link into this issue would be ok) to explain why this has been added since it's not obvious why we've added this only for C6.